common/eager_offset.ts

```diff
@@ -0,0 +1,24 @@
+// # Why
+// We need this to allow global variables.
+//
+// # How
+// This is a hacky workaround to make the lazy `offset` variable of
+// the AS stub runtime be eagerly executed. It works by calling the
+// `__alloc` function which uses the `offset`, so that its evaluated.
+//
+// Since we only call `__alloc` on the Rust (graph-node) side, the
+// `offset` variable isn't evaluated, so if you compile the AS code
+// without this file, the global vars stay after its declaration on
+// the `_start` function of the generated `.wasm` file. This causes
+// problems in the following allocations.
```

Comment:
I think I'd have a hard time understanding this paragraph if I didn't already know what this is. Maybe this can be phrased more clearly.

Reply:
Ok, I'll try to rephrase it!

Reply:
Fixed!